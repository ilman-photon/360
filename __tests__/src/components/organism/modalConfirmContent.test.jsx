import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import ModalConfirmContent from "../../../../src/components/organisms/ScheduleAppointment/ModalScheduling/modalConfirmContent";

describe("App", () => {
    let container;
    beforeEach(() => {
      container = render(<ModalConfirmContent />);
    });
  
    it("ModalConfirmContent render", () => {
        expect(container).toMatchSnapshot();
    });
    
  });
  
