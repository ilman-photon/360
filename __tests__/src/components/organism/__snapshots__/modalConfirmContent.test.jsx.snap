// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App ModalConfirmContent render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="boxModalContents MuiBox-root css-17wkslb"
      >
        <h2
          aria-label="Appointment confirmation page"
          class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-aw0cy1-MuiTypography-root-MuiDialogTitle-root"
          id="customized-dialog-title"
        >
          <span
            arialabel="You’re Scheduled!"
            class="MuiTypography-root MuiTypography-bodyMedium scheduledText css-37jdci-MuiTypography-root"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yziclw-MuiSvgIcon-root"
              data-testid="CheckCircleRoundedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM9.29 16.29 5.7 12.7a.9959.9959 0 0 1 0-1.41c.39-.39 1.02-.39 1.41 0L10 14.17l6.88-6.88c.39-.39 1.02-.39 1.41 0 .39.39.39 1.02 0 1.41l-7.59 7.59c-.38.39-1.02.39-1.41 0z"
              />
            </svg>
             
            You’re Scheduled!
          </span>
          <button
            aria-label="close option"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1ehv1gf-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="CloseIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </h2>
        <div
          class="MuiDialogContent-root css-1phsj2c-MuiDialogContent-root"
        >
          <div
            aria-label="nono button"
            class="registeredUsernameWrapper"
            sx="[object Object]"
          >
            <div
              aria-label="thanksBar"
              class="thanksBar MuiBox-root css-1bwq3er"
              tabindex="0"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1qquyr5-MuiSvgIcon-root"
                data-testid="MailOutlineIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
                />
              </svg>
               
              thanksBar
            </div>
          </div>
          <div
            class="bottomParagraph"
          >
            <a
              aria-label="isEmergency"
              arialabel="sIf this is a medical emergency, please call 911"
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
              data-mui-internal-clone-element="true"
              data-testid="loginlink"
              role="link"
              tabindex="0"
            >
              <span
                class="medicLink"
              >
                isEmergency
              </span>
            </a>
          </div>
          <div
            class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root cardDate css-1v5z4dq-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-1aithki-MuiCardContent-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 dateBold css-eyz4cw-MuiTypography-root"
              >
                -
              </p>
              <div
                style="display: inline-flex;"
              >
                <button
                  class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root addCalendarButton css-vij0pv-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  <p
                    aria-label="Add to calendar"
                    class="MuiTypography-root MuiTypography-body1 css-v8btcj-MuiTypography-root"
                  >
                    <svg
                      aria-hidden="false"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-q0n8nh-MuiSvgIcon-root"
                      data-testid="CalendarTodayIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                      />
                    </svg>
                     
                    Add to calendar
                  </p>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
              <p
                aria-label="Purpose of Visit"
                class="MuiTypography-root MuiTypography-body1 dateBold css-ahj2mt-MuiTypography-root"
              >
                Purpose of Visit
              </p>
              <p
                aria-label="Eye exam"
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Eye exam
              </p>
            </div>
          </div>
          <div
            class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-4bhyjd-MuiCardContent-root"
            >
              <div
                class="css-1p5q5e5-MuiStack-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container css-zmgw4q-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root css-vj1n65-MuiGrid-root"
                  >
                    <div
                      class="appointment MuiBox-root css-1qa5ro4"
                    >
                      <div
                        class="displayFlex MuiBox-root css-0"
                      >
                        <div
                          class="imageContainer MuiBox-root css-0"
                        >
                          <svg
                            alt="Doctor Image"
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium profilePhoto css-epjost-MuiSvgIcon-root"
                            data-testid="appointment_provider_profile_image"
                            focusable="false"
                            tabindex="0"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                            />
                          </svg>
                        </div>
                        <div
                          class="bioContainer  MuiBox-root css-1oovyp6"
                        >
                          <h2
                            class="MuiTypography-root MuiTypography-h2 doctorNameAppointment css-1i5ehwd-MuiTypography-root"
                            data-testid="appointment_provider_profile_name"
                            tabindex="0"
                          />
                          <h3
                            class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
                            tabindex="0"
                          >
                            Scripps Eyecare
                          </h3>
                          <div
                            class="detailContainer MuiBox-root css-0"
                          >
                            <div
                              class="MuiBox-root css-0"
                            >
                              <div
                                aria-label="Doctor Address"
                                class="MuiBox-root css-0"
                              >
                                <p
                                  class="MuiTypography-root MuiTypography-body2 address  css-166djn1-MuiTypography-root"
                                  tabindex="0"
                                />
                              </div>
                              <div
                                class="phoneContainer MuiBox-root css-0"
                              >
                                <a
                                  class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways phoneLink css-1ps4owl-MuiTypography-root-MuiLink-root"
                                  tabindex="0"
                                />
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="getDirectionLink MuiBox-root css-0"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="DirectionsOutlinedIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="m22.43 10.59-9.01-9.01c-.75-.75-2.07-.76-2.83 0l-9 9c-.78.78-.78 2.04 0 2.82l9 9c.39.39.9.58 1.41.58.51 0 1.02-.19 1.41-.58l8.99-8.99c.79-.76.8-2.02.03-2.82zm-10.42 10.4-9-9 9-9 9 9-9 9zM8 11v4h2v-3h4v2.5l3.5-3.5L14 7.5V10H9c-.55 0-1 .45-1 1z"
                        />
                      </svg>
                      <a
                        aria-label="Get directions"
                        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways getDirectionLinkText css-1ps4owl-MuiTypography-root-MuiLink-root"
                        role="link"
                        tabindex="0"
                      >
                        Get directions
                      </a>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root cardPatient css-1v5z4dq-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-1qboppe-MuiCardContent-root"
            >
              <p
                aria-label="Patient Information heading"
                aria-roledescription="Heading"
                class="MuiTypography-root MuiTypography-body1 patientBoxLabel css-z2eky3-MuiTypography-root"
              >
                patientInformation
              </p>
              <div
                class="css-nen11g-MuiStack-root"
              >
                <div
                  class="labelContainer"
                  style="justify-content: unset;"
                >
                  <div
                    class="label"
                    role="text"
                    style="color: rgb(0, 59, 74); font-size: 16px;"
                    tabindex="0"
                  >
                    Name
                  </div>
                </div>
                <div
                  class="css-1sbz1s3-MuiStack-root"
                >
                  <div>
                    <span
                      class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                    >
                      -
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="bottomParagraph"
          >
            <span
              aria-label="Already have an account? Sign in"
              class="MuiTypography-root MuiTypography-caption css-18ywxbx-MuiTypography-root"
              tabindex="0"
            >
              Already have an account?
               
              <a
                class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                data-testid="loginlink"
                href="/patient/login"
              >
                <a
                  class="loginLink"
                >
                  Sign in
                </a>
              </a>
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="boxModalContents MuiBox-root css-17wkslb"
    >
      <h2
        aria-label="Appointment confirmation page"
        class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-aw0cy1-MuiTypography-root-MuiDialogTitle-root"
        id="customized-dialog-title"
      >
        <span
          arialabel="You’re Scheduled!"
          class="MuiTypography-root MuiTypography-bodyMedium scheduledText css-37jdci-MuiTypography-root"
          tabindex="0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yziclw-MuiSvgIcon-root"
            data-testid="CheckCircleRoundedIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM9.29 16.29 5.7 12.7a.9959.9959 0 0 1 0-1.41c.39-.39 1.02-.39 1.41 0L10 14.17l6.88-6.88c.39-.39 1.02-.39 1.41 0 .39.39.39 1.02 0 1.41l-7.59 7.59c-.38.39-1.02.39-1.41 0z"
            />
          </svg>
           
          You’re Scheduled!
        </span>
        <button
          aria-label="close option"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1ehv1gf-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="CloseIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </h2>
      <div
        class="MuiDialogContent-root css-1phsj2c-MuiDialogContent-root"
      >
        <div
          aria-label="nono button"
          class="registeredUsernameWrapper"
          sx="[object Object]"
        >
          <div
            aria-label="thanksBar"
            class="thanksBar MuiBox-root css-1bwq3er"
            tabindex="0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1qquyr5-MuiSvgIcon-root"
              data-testid="MailOutlineIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
              />
            </svg>
             
            thanksBar
          </div>
        </div>
        <div
          class="bottomParagraph"
        >
          <a
            aria-label="isEmergency"
            arialabel="sIf this is a medical emergency, please call 911"
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
            data-mui-internal-clone-element="true"
            data-testid="loginlink"
            role="link"
            tabindex="0"
          >
            <span
              class="medicLink"
            >
              isEmergency
            </span>
          </a>
        </div>
        <div
          class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root cardDate css-1v5z4dq-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-1aithki-MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 dateBold css-eyz4cw-MuiTypography-root"
            >
              -
            </p>
            <div
              style="display: inline-flex;"
            >
              <button
                class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root addCalendarButton css-vij0pv-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                <p
                  aria-label="Add to calendar"
                  class="MuiTypography-root MuiTypography-body1 css-v8btcj-MuiTypography-root"
                >
                  <svg
                    aria-hidden="false"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-q0n8nh-MuiSvgIcon-root"
                    data-testid="CalendarTodayIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                    />
                  </svg>
                   
                  Add to calendar
                </p>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
            <p
              aria-label="Purpose of Visit"
              class="MuiTypography-root MuiTypography-body1 dateBold css-ahj2mt-MuiTypography-root"
            >
              Purpose of Visit
            </p>
            <p
              aria-label="Eye exam"
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Eye exam
            </p>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-4bhyjd-MuiCardContent-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-zmgw4q-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                >
                  <div
                    class="appointment MuiBox-root css-1qa5ro4"
                  >
                    <div
                      class="displayFlex MuiBox-root css-0"
                    >
                      <div
                        class="imageContainer MuiBox-root css-0"
                      >
                        <svg
                          alt="Doctor Image"
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium profilePhoto css-epjost-MuiSvgIcon-root"
                          data-testid="appointment_provider_profile_image"
                          focusable="false"
                          tabindex="0"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                          />
                        </svg>
                      </div>
                      <div
                        class="bioContainer  MuiBox-root css-1oovyp6"
                      >
                        <h2
                          class="MuiTypography-root MuiTypography-h2 doctorNameAppointment css-1i5ehwd-MuiTypography-root"
                          data-testid="appointment_provider_profile_name"
                          tabindex="0"
                        />
                        <h3
                          class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
                          tabindex="0"
                        >
                          Scripps Eyecare
                        </h3>
                        <div
                          class="detailContainer MuiBox-root css-0"
                        >
                          <div
                            class="MuiBox-root css-0"
                          >
                            <div
                              aria-label="Doctor Address"
                              class="MuiBox-root css-0"
                            >
                              <p
                                class="MuiTypography-root MuiTypography-body2 address  css-166djn1-MuiTypography-root"
                                tabindex="0"
                              />
                            </div>
                            <div
                              class="phoneContainer MuiBox-root css-0"
                            >
                              <a
                                class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways phoneLink css-1ps4owl-MuiTypography-root-MuiLink-root"
                                tabindex="0"
                              />
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="getDirectionLink MuiBox-root css-0"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="DirectionsOutlinedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="m22.43 10.59-9.01-9.01c-.75-.75-2.07-.76-2.83 0l-9 9c-.78.78-.78 2.04 0 2.82l9 9c.39.39.9.58 1.41.58.51 0 1.02-.19 1.41-.58l8.99-8.99c.79-.76.8-2.02.03-2.82zm-10.42 10.4-9-9 9-9 9 9-9 9zM8 11v4h2v-3h4v2.5l3.5-3.5L14 7.5V10H9c-.55 0-1 .45-1 1z"
                      />
                    </svg>
                    <a
                      aria-label="Get directions"
                      class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways getDirectionLinkText css-1ps4owl-MuiTypography-root-MuiLink-root"
                      role="link"
                      tabindex="0"
                    >
                      Get directions
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root cardPatient css-1v5z4dq-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-1qboppe-MuiCardContent-root"
          >
            <p
              aria-label="Patient Information heading"
              aria-roledescription="Heading"
              class="MuiTypography-root MuiTypography-body1 patientBoxLabel css-z2eky3-MuiTypography-root"
            >
              patientInformation
            </p>
            <div
              class="css-nen11g-MuiStack-root"
            >
              <div
                class="labelContainer"
                style="justify-content: unset;"
              >
                <div
                  class="label"
                  role="text"
                  style="color: rgb(0, 59, 74); font-size: 16px;"
                  tabindex="0"
                >
                  Name
                </div>
              </div>
              <div
                class="css-1sbz1s3-MuiStack-root"
              >
                <div>
                  <span
                    class="MuiTypography-root MuiTypography-bodyMedium css-1oitrp4-MuiTypography-root"
                  >
                    -
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="bottomParagraph"
        >
          <span
            aria-label="Already have an account? Sign in"
            class="MuiTypography-root MuiTypography-caption css-18ywxbx-MuiTypography-root"
            tabindex="0"
          >
            Already have an account?
             
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
              data-testid="loginlink"
              href="/patient/login"
            >
              <a
                class="loginLink"
              >
                Sign in
              </a>
            </a>
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
